mlir/include/mlir/Dialect/LLVMIR/ROCDLOps.td

```diff
@@ -516,7 +516,7 @@ def ROCDL_RawBufferAtomicCmpSwap :
 }
 
 //===---------------------------------------------------------------------===//
-// MI-100 and MI-200 buffer atomic floating point add intrinsic
+// MI-100, MI-200 and MI-300 global/buffer atomic floating point add intrinsic
```

Comment:
nit: at some point, we should start using architecture names (gfx9, etc.) to match llvm architecture models better

Reply:
Yeah